pages/entropy/error-codes.mdx

```diff
@@ -0,0 +1,21 @@
+| ----------- | --------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 0xd82dd966  | AssertionFailure()          | Contract invariant failed.                                                                                                                                                                                                  |
+| 0xda041bdf  | ProviderAlreadyRegistered() | Provider already registered.                                                                                                                                                                                                |
+| 0xdf51c431  | NoSuchProvider()            | Requested Provider does not exist.                                                                                                                                                                                          |
+| 0xc4237352  | NoSuchRequest()             | Request does not exist.                                                                                                                                                                                                     |
+| 0x3e515085  | OutOfRandomness()           | Provider is our of committed random numbers.                                                                                                                                                                                |
+| 0x025dbdd4  | InsufficientFee()           | Transaction fee insufficient.                                                                                                                                                                                               |
+| 0xb8be1a8d  | IncorrectRevelation()       | Revelation does not match commitment.                                                                                                                                                                                       |
+| 0xb463ce7a  | InvalidUpgradeMagic()       | Governance message is invalid.                                                                                                                                                                                              |
+| 0x82b42900  | Unauthorized()              | `msg.sender` is not allowed to invoke this method.                                                                                                                                                                          |
+| 0x92555c0e  | BlockhashUnavailable()      | Blockhash is 0.                                                                                                                                                                                                             |
```

Comment:
If entropy request was made using the `useBlockhash` flag and the reveal method is called too late, the blockhash is not available anymore and the request can not be fulfilled.

Reply:
That's too verbose. That should be added in a troublshooting page.